tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -28,9 +28,9 @@ the instructions to your particular needs.  The names of the pip and Docker
 images are listed in the corresponding installation sections.
 
 If you encounter installation errors, see
-[common problems](#common_install_problems) for some solutions.
+[common problems](#common-installation-problems) for some solutions.
 
-## Pip Installation {#pip_install}
```

Comment:
Is it wise to simply remove the {#anchor} notation? From what I understand, this syntax is used internally to render the Markdown to HTML with id tags, which is then used on the tensorflow.org website.

If the goal is to have this work on both Github and tensorflow.org, my suggestion is, rather than removing the {#anchor}, replace {#old-anchor} with the {#new-anchor}, such that "#new-anchor" matches the automatically created id inside of Github. 

for example, I suggest that this line should become:
`## Pip Installation {#pip_installation}`